docs/upgrades.md

```diff
@@ -38,19 +38,26 @@ sequenceDiagram
        A->>A: Extract new Agent artifact
        A->>A: Replace current Agent artifact with new one
        A->>UM: Create
+       A->>A: Update active commit file
        A->>UW: Start
        A->>A: Rexec to start new Agent artifact
        A->>FS: Ack successful upgrade
-       UW->>UM: Remove
        FS->>ES: Write successful ack in `.fleet-actions-results`
        FS->>ES: Update Agent doc in `.fleet-agents`<br />set `upgrade_status` = null<br />`upgraded_at` = <now><br />`upgrade_started_at` = null
        UI->>UI: Show Agent status as "healthy"
-   end
-   opt Rollback
-       UW->>A: Start
-       A->>FS: Ack failed upgrade
-       FS->>ES: Update Agent doc in `.fleet-agents`<br />set `upgrade_status` = null<br />`upgraded_at = <now>
-       UI->>UI: Show Agent status as "healthy"
-       UW->>UM: Remove
-   end
+       UW->>UW: Start watching new Agent
+       alt New Agent is OK
+         UW->>UM: Remove
+         UW->>UW: Cleanup old Agent files
+       else Rollback
+         UW->>UW: Replace current Agent artifact with old one
```

Comment:
This is just a change to the symlink correct? Is it worth clarifying this to specifically mention the word symlink? 

This could read as if it is doing something involved by manipulating the extracted agent, but it just leaves that alone.

The same applies to the same wording on Line 39 above.

Reply:
Yes, you're correct.  I don't know why I decided to use the less concrete wording here instead 🤷.  Will fix in both places.  Thanks.

Reply:
Fixed in b2084030f37c0f161e645a4792153f2deb28e647.